launcher/InstanceImportTask.cpp

```diff
@@ -45,38 +45,41 @@
 #include "icons/IconList.h"
 #include "icons/IconUtils.h"
 
+#include "modplatform/ModIndex.h"
 #include "modplatform/flame/FlameInstanceCreationTask.h"
+#include "modplatform/helpers/GetModPackExtraInfoTask.h"
 #include "modplatform/modrinth/ModrinthInstanceCreationTask.h"
 #include "modplatform/technic/TechnicPackProcessor.h"
 
 #include "settings/INISettingsObject.h"
+<<<<<<< HEAD
+#include "tasks/MultipleOptionsTask.h"
+    == == ==
+    =
 #include "tasks/Task.h"
+        >>>>>>> 42b06674bba9b0d7d7b52189d0693eca82ba5a13
```

Comment:
```suggestion
#include "tasks/MultipleOptionsTask.h"
#include "tasks/Task.h"
```

merge conflict left in